src/test/java/teammates/test/cases/automated/FeedbackSessionEmailTaskQueueTest.java

```diff
@@ -254,4 +254,27 @@ public void testScheduleFeedbackSessionPublishedEmails() throws Exception {
             assertEquals(FeedbackSessionsEmailTaskQueueCallback.taskCount, 0);
         }
     }
+    
+    @Test
+    public void testSendFeedbackSessionUnpublishedEmail() throws Exception {
+        FeedbackSessionsEmailTaskQueueCallback.resetTaskCount();
+        FeedbackSessionsLogic fsLogic = FeedbackSessionsLogic.inst();
+
+        ______TS("1 unpublished email to be sent");
+        
+        FeedbackSessionAttributes fsa = dataBundle.feedbackSessions.get("session2InCourse1");
+        int counter = 0;
+
+        while (counter != 10) {
```

Comment:
Add a comment to explain the purpose of this code block. It is not immediately clear to the reader what's going on.